test/conversion.test.js

```diff
@@ -27,6 +27,52 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	describe('[Github #2] - set the method to ', function() {
+		it('GET if --get option is given in the curl command', function(done) {
+			convert({
+				type: 'string',
+				data: 'curl -d "key=example" --get https://example.com'
```

Comment:
We don't need a -d parameter here.The test only mentions that GET would be the request method if --get is provided.